products.json

```diff
@@ -1,4 +1,18 @@
 [
+    {
+        "name": "spark-3.5.1",
```

Comment:
@welpaolo should we call this Spark 3.5 and also change the name of the other one from "spark" to "spark-3.4"?

Does this name matter?

Reply:
Renamed to `spark-3.5` and `spark-3.4`.